build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -33,6 +33,12 @@ rules:
       - pods/status
     verbs:
       - patch
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
```

Comment:
This is a high-risk permission we mustn't grant to agents. CVEs always come after it.

If accessing secret to get file server's credential is must, you could define a well-known secret and document users to put credentials there. Then only grant a single secret's permission to agent.

Reply:
Sure, that's reasonable. Is `default` namespace recommended ? i will start from this.

Reply:
I think we should use kube-system as other well-known secrets are in it too. It's also a special namespace normal users can't access, so the secret wouldn't be exposed to normal users.

Reply:
Updated to use a dedicated secret. And validate the name in webhook.